Sources/Private/Model/Layers/LayerModel.swift

```diff
@@ -93,7 +93,7 @@ class LayerModel: Codable, DictionaryInitializable {
     inFrame = try container.decode(Double.self, forKey: .inFrame)
     outFrame = try container.decode(Double.self, forKey: .outFrame)
     startTime = try container.decode(Double.self, forKey: .startTime)
-    transform = try container.decode(Transform.self, forKey: .transform)
+    transform = try container.decodeIfPresent(Transform.self, forKey: .transform) ?? .default
```

Comment:
All of the individual properties in a `Transform` are optional and have default values, so we can easily make the `Transform` itself optional with a default